mithril-aggregator/tests/create_certificate.rs

```diff
@@ -127,6 +140,76 @@ async fn create_certificate() {
         )
     );
 
+    comment!(
```

Comment:
Maybe this could be done in a dedicated integration test specific to chain rollbacks?

Reply:
After discussing together we will put this subject on hold to include it in a more global reflection on the integration tests (where to draw the lines between them on other tests ? etc ...).